instruments/toptica/topmode.py

```diff
@@ -137,11 +139,23 @@ def enable(self):
         @enable.setter
         def enable(self, newval):
             if not isinstance(newval, bool):
-                raise TypeError(
-                    "Laser emmission must be a boolean, got: {}".format(newval))
+                raise TypeError("Emission status must be a boolean, got: "
+                                "{}".format(type(newval)))
+            if not self.is_connected:
+                return
             self.parent.set(self.name + ":enable-emission", newval)
 
         @property
+        def is_connected(self):
```

Comment:
What about the case when the laser was initially connected, but then is later disconnected? It should timeout during the `self.serial_number` call. We might not want to raise an exception in that case, but maybe return `False`? I guess it depends what kind of "connected" we are going for here. Physically and/or logically.